controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -973,6 +975,13 @@ func (r *ReconcileHyperConverged) setLabels(req *common.HcoRequest) {
 	}
 }
 
+func (r *ReconcileHyperConverged) setMetrics(req *common.HcoRequest) {
```

Comment:
The name of this method is too general. We set many metrics, but here we set a specific one.

Reply:
Also, is settinh the metric the main pupose of the function, or checking the memory?

Reply:
@nunnatsa The best I could think of was `setHcoCrMetrics` but if you have a better idea I am all for it.
Main purpose of the function is to set the memory overcommit metric, I can extract all the logic into `doReconcile` and use `metrics.SetMemoryOvercommitPercentage` directly from there if it makes more readability sense.

Reply:
@nunnatsa I've removed the function, now i set the metric directly from doReconcile. Please advice regarding a better place.